docker/client/config/default.conf

```diff
@@ -2,23 +2,37 @@ server {
     listen 80;
     server_name localhost;
 
-    root    /usr/share/nginx/html;
+    root /usr/share/nginx/html;
     index index.html;
 
+    # Static files with compression caching
+    location ~* \.(whl|js|css|json|wasm)$ {      
```

Comment:
I'm on the fence about no-cache here 🤔.  We want caching when the file has changed, but not otherwise.

Reply:
The whl files from Pyodide should be versioned, but I don't think the Quadratic one is.